llvm/test/SYCLLowerIR/esimd_global.ll

```diff
@@ -6,18 +6,18 @@
 target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
 target triple = "spir64-unknown-unknown-sycldevice"
 
-%"class._ZTSN2cl4sycl5INTEL3gpu4simdIiLi16EEE.cl::sycl::INTEL::gpu::simd" = type { <16 x i32> }
+%"class._ZTSN2cl4sycl5INTEL3gpu4simdIiLi16EEE.cl::sycl::ext::intel::gpu::simd" = type { <16 x i32> }
```

Comment:
Here and in all other similar places. Correct namespace for public esimd apis is `sycl::ext::intel::experimental::esimd`. E.g.
`sycl::ext::intel::experimental::esimd::simd`. My suggestion is either not to touch the .ll files at all, or change to "namespace-neutral" form like `%"class.esimd::simd_IiLi16"`. Note the `IiLi16` stem, which is needed to unique from other simd class instantiations within LLVMIR.

Reply:
@kbobrovs are you ok to address all your comments in the separate PR due to upcoming internal feature freeze? The purpose of this feature was to make a transition from INTEL/ONEAPI namespaces to ext::intel/oneapi w/o changing any logic.

Reply:
@dm-vodopyanov - you mean dropping ESIMD changes from  this PR, and creating a new one? Yes, I'm OK, if so.

Reply:
No, I meant to commit PR as is to have time to cherry-pick to internal repo, and apply comments as separate patch and also cherry-pick it but after feature freeze.

Anyway, I already applied comments.

Reply:
Reverted such changes in *.ll files